docs/examples/fa-examples.md

```diff
@@ -24,76 +39,113 @@ my_dfa = DFA(
     final_states={'q1'}
 )
 
-try:
-    while True:
-        if my_dfa.accepts_input(input('Please enter your input: ')):
-            print('Accepted')
-        else:
-            print('Rejected')
-except KeyboardInterrupt:
-    print('')
+read_user_input(my_dfa)
+```
+
+We can also do the same with an NFA we define. Note that the
+transition dictionary for the NFA has a different structure than
+that of the DFA, and that we are working over a different input
+alphabet than the previous example:
+
+```python
+from automata.fa.nfa import NFA
+
+# NFA which matches strings beginning with "a", ending with "a", and
+# containing no consecutive "b"s
+my_nfa = NFA(
+    states={"q0", "q1", "q2"},
+    input_symbols={"a", "b"},
+    transitions={
+        "q0": {"a": {"q1"}},
+        "q1": {"a": {"q1"}, "": {"q2"}},
+        "q2": {"b": {"q0"}},
+    },
+    initial_state="q0",
+    final_states={"q1"},
+)
+
+read_user_input(my_nfa)
 ```
 
 ## Subset for NFAs
 
 The `NFA` does not have a built-in method for checking whether it is a subset
```

Comment:
@eliotwrobson Can you please remind me the reason we don't have an `issubset` method for NFAs? I feel like it was an intentional decision we made earlier this year, but I can't remember why.

Reply:
It would require creating a new NFA in-memory. In hindsight, maybe a little overkill to worry about, I think the overhead on performance wouldn't be too bad.

Reply:
@eliotwrobson Fair enough! We can revisit that discussion later.